tensorflow/core/graph/gpu_fusion_pass.cc

```diff
@@ -316,7 +341,9 @@ Status ROCmFusionPass::Run(const GraphOptimizationPassOptions& options) {
 }
 
 bool ROCmFusionPass::RunPass(Graph* graph) {
-  // DumpGraph("Before running ROCmFusionPass", &**g);
+  if (getenv("TF_ROCM_FUSION_DUMP_GRAPH_BEFORE")) {
```

Comment:
@deven-amd also please help enumerate env vars introduced in this PR in the description so other users can leverage them effectively.